elasticdl/python/ps/parameters.py

```diff
@@ -0,0 +1,49 @@
+import tensorflow as tf
+
+from elasticdl.python.common.ndarray import tensor_to_ndarray
+from elasticdl.python.ps.embedding_table import create_embedding_table
+
+
+class Parameters(object):
+    def __init__(self):
+        self.init_status = False
+        self.non_embedding_params = {}
+        self.embedding_params = {}
```

Comment:
Is this ElasticDL embedding params or Keras native embedding params?

Reply:
It's ElasticDL embedding params. The key is an embedding layer name, the value is a n`EmbeddingTable` object. I will add some comments in the class `Parameters`.